.build/tasks/New-Release.GitHub.build.ps1

```diff
@@ -141,9 +144,9 @@ task Create_ChangeLog_GitHub_PR -if ($GitHubToken) {
         }
     }
 
-    git pull origin master --tag
-    # Look at the tags on latest commit for origin/master (assume we're on detached head)
-    $TagsAtCurrentPoint = git tag -l --points-at (git rev-parse origin/master)
+    git pull origin $MainGitBranch --tag
+    # Look at the tags on latest commit for origin/$MainGitBranch (assume we're on detached head)
+    $TagsAtCurrentPoint = git tag -l --points-at (git rev-parse origin/$MainGitBranch)
```

Comment:
when we use $MainGitBranch within what's a native command, we should stringify it:
`$TagsAtCurrentPoint = git @('tag', '-l', '--points-at', $(git @('rev-parse', "origin/$MainGitBranch"))`

Reply:
there's other places we should do that at some point (not this PR though)

Reply:
Done!